.travis/smoke-test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+function finish {
+    make logs stop
+}
+
+trap finish EXIT
+
+make start
+
+.scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME:-article-store}_app_1"
```

Comment:
Turns out `_1` isn't true.

Reply:
The `docker-compose` version on Travis CI had a bug related to this recently:
https://github.com/libero/jats-ingester/pull/56

Reply:
7abe9f5ea3c108f83a70f2e7de2dd00d8185c636 avoids guessing.